pkg/kn/commands/service/service_list_test.go

```diff
@@ -78,6 +91,31 @@ func TestServiceListDefaultOutput(t *testing.T) {
 	assert.Check(t, util.ContainsAll(output[2], "bar", "bar.default.example.com", "2"))
 }
 
+func TestServiceGetOneOutput(t *testing.T) {
+	service := createMockServiceWithParams("foo", "foo.default.example.com", 1)
+	serviceList := &v1alpha1.ServiceList{Items: []v1alpha1.Service{*service}}
+	action, output, err := fakeServiceList([]string{"service", "list", "foo"}, serviceList)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if action == nil {
+		t.Errorf("No action")
+	} else if !action.Matches("list", "services") {
+		t.Errorf("Bad action %v", action)
+	}
+	assert.Check(t, util.ContainsAll(output[0], "NAME", "DOMAIN", "GENERATION", "AGE", "CONDITIONS", "READY", "REASON"))
+	assert.Check(t, util.ContainsAll(output[1], "foo", "foo.default.example.com", "1"))
+}
+
+func TestServiceGetWithTwoSrvName(t *testing.T) {
+	service := createMockServiceWithParams("foo", "foo.default.example.com", 1)
+	serviceList := &v1alpha1.ServiceList{Items: []v1alpha1.Service{*service}}
+	_, _, err := fakeServiceList([]string{"service", "list", "foo", "bar"}, serviceList)
+	if err != nil {
```

Comment:
I think we should use https://godoc.org/gotest.tools/assert#ErrorContains without any conditional as error is expected in this case.